data/Shashankzz0007.json

```
@@ -0,0 +1,62 @@
+{
+  "name": "Shashank Deshpande",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Open Source Enthusiast! | DevOps | MERN Developer | Self-paced Full-Stack Developer",
+  "avatar": "https://github.com/Shashankzz0007.png",
+  "tags": ["Open Source", "javascript", "typescript", "Reactjs", "DevOps"],
```

Not sure tags can contain spaces or CAPS.
```suggestion
  "tags": ["opensource", "javascript", "typescript", "reactjs", "devops"],
```